Write some basic data modules

adamwight opened the pull request:
FIXME:
* apply linter
* data location needs to be out-of-repo

ahn-nath commented on wiki_article_count.py:
```diff
@@ -0,0 +1,37 @@
+
+
+@cached("wiki_article_count.csv")
+def get_wiki_counts():
```
Interesting example of how to get the article count by Wikipedia language edition.

adamwight replied:
Now I'm leaning towards a port of the other repository, so that the library here is capable of regenerating fresh statistics.

adamwight replied:
(done and pushed to branch)

ahn-nath commented:
@adamwight 
Can you clarify if the FIXME section of your description are pending items for this pull request?

adamwight committed:
Ship requirements.txt

adamwight committed:
Module to list all active wikipedias

Note that we're filtering out inactive, "closed" editions.

adamwight committed:
Port wiki article counter

This provides more data than
https://gitlab.com/wmde/technical-wishes/wiki-article-counter/

adamwight committed:
Module to fetch data from the ContentTranslation API.

adamwight commented:
Let's say these FIXMEs aren't blockers.  Linting will be its own interesting PR, and our team has been leaning towards a separate store for data but this is just a matter of configuration and doesn't need to involve the code here.
